optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/movecountperstep/MoveCountPerStepSubSingleStatistic.java

```diff
@@ -42,8 +42,8 @@ public MoveCountPerStepSubSingleStatistic(SubSingleBenchmarkResult subSingleBenc
     @Override
     public void open(StatisticRegistry registry, Tags runTag, Solver<Solution_> solver) {
         registry.addListener(SolverMetric.MOVE_COUNT_PER_STEP, timeMillisSpent -> {
-            registry.getGaugeValue(SolverMetric.MOVE_COUNT_PER_STEP + ".accepted", runTag, accepted -> {
-                registry.getGaugeValue(SolverMetric.MOVE_COUNT_PER_STEP + ".accepted", runTag, selected -> {
+            registry.getGaugeValue(SolverMetric.MOVE_COUNT_PER_STEP.getMeterId() + ".accepted", runTag, accepted -> {
+                registry.getGaugeValue(SolverMetric.MOVE_COUNT_PER_STEP.getMeterId() + ".selected", runTag, selected -> {
```

Comment:
Since you control the registry API, I suggest you continue accepting the enum and do the `getMeterId()` call internally. Less repetition.